datamodel/include/datamodel/math/Matrix.hpp

```diff
@@ -384,8 +385,8 @@ rcbe::math::Matrix<V, 4, 4> operator*(const rcbe::math::Matrix<V, 4, 4> &lhs, co
 
     return rcbe::math::Matrix<V, 4, 4> {interm};
 }
-rcbe::math::Matrix4x4 operator+(const rcbe::math::Matrix4x4 &lhs, const rcbe::math::Matrix4x4 &rhs);
+R_PUBLIC_API rcbe::math::Matrix4x4 operator+(const rcbe::math::Matrix4x4 &lhs, const rcbe::math::Matrix4x4 &rhs);
 
-rcbe::math::Vector4d operator*(const rcbe::math::Matrix4x4 &lhs, const rcbe::math::Vector4d &rhs);
+R_PUBLIC_API rcbe::math::Vector4d operator*(const rcbe::math::Matrix4x4 &lhs, const rcbe::math::Vector4d &rhs);
```

Comment:
Same about prefixes, just mark the whole class, it should be enough.

Reply:
This function are not calss members, so this particular prefixes, should stay.